vars/runIntegTests.groovy

```diff
@@ -0,0 +1,24 @@
+void call(Map args = [:]) {
+    git url: 'https://github.com/opensearch-project/opensearch-build.git', branch: 'main'
+
+    def lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+    String testManifest = args.testManifest ?: "manifests/${TEST_MANIFEST}"
+
+    String tarball = args.architecture == "x64" ? env.ARTIFACT_URL_linux_x64 : env.ARTIFACT_URL_linux_arm64
```

Comment:
I think a proper `if` or `case` is better here to handle `x64` vs. `arm64` vs. something else that doesn't work to avoid suprises.